provisioner/docker/config_centos-8.yaml

```diff
@@ -0,0 +1,24 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+docker:
+        memory_limit: "4g"
+        image: "bigtop/puppet:trunk-centos-8"
+
+repo: "http://repos.bigtop.apache.org/releases/1.5.0/centos/8/$basearch"
```

Comment:
Is this URL invalid before the release of 1.5.0? I got the the following error.
```
Error: Execution of '/usr/bin/dnf -d 0 -e 1 -y install java-1.8.0-openjdk-devel' returned 1: Error: Failed to download metadata for repo 'Bigtop_0'
Error: /Stage[main]/Jdk/Package[jdk]/ensure: change from 'purged' to 'present' failed: Execution of '/usr/bin/dnf -d 0 -e 1 -y install java-1.8.0-openjdk-devel' returned 1: Error: Failed to download metadata for repo 'Bigtop_0'
```

Reply:
That's right, just the same as [config_fedora-31.yaml](/apache/bigtop/blob/master/provisioner/docker/config_fedora-31.yaml). I replaced that URL with the existing one temporally when I tested it.